src/forms/FormErrorMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+import "./FormErrorMessage.scss"
+
+export interface FormErrorMessageProps {
+  /** Message content */
+  children: React.ReactNode
+  /** Element ID */
+  id?: string
+  /** Additional CSS classes */
+  className?: string
+}
+
+/**
+ * Component used to display an error message under a form field when the user input is invalid.
+ */
+const FormErrorMessage = (props: FormErrorMessageProps) => {
+  const classNames = ["form-error-message"]
+  if (props.className) classNames.push(props.className)
+
+  return (
+    <span id={props.id} className={classNames.join(" ")} aria-live="assertive">
+      {props.children}
+    </span>
+  )
+}
+
+export default FormErrorMessage
```

Comment:
Should we also export the non-default version the same way we do in ui-components?
`export { FormErrorMessage as default, FormErrorMessage }`

Also, I believe you will need to export it in index.ts so that we can import it from `@bloom-housing/ui-seeds` in bloom

Reply:
@ludtkemorgan I've been looking at the Vibe system by Monday, and I like how they do it…only default exports from the individual files, but then in an index file it looks something like this:

```js
export { default as Dialog } from "./Dialog/Dialog";
export { default as Tooltip } from "./Tooltip/Tooltip";
export { default as Button } from "./Button/Button";
```

I can update this PR along similar lines if that sounds good to you.

Reply:
That sounds good!